src/lib/utils/helpers.ts

```diff
@@ -170,3 +170,7 @@ export function buildResurrectionDateString(
     msUntilResurrection < 0 ? `${humanizedDuration} ago` : humanizedDuration;
   return `${resurrectionDateString} (${timeUntilResurrection})`;
 }
+
+export function isBytes32(value: string): boolean {
+  return value.length === 66 && value.startsWith('0x') && /^[0-9a-fA-F]+$/.test(value.slice(2));
```

Comment:
Do we require the private keys start with `0x`? We should either make this not a requirement (detect if it starts with `0x` and if not, add it then validate), or add some text somewhere on the page saying that is a requirement.

Reply:
Best option would probably be to accept both with the `0x` or without.